clients/lsp-javascript.el

```diff
@@ -72,6 +72,8 @@
                                     (warn (concat "The javascript-typescript-langserver (jsts-ls) is unmaintained; "
                                                   "it is recommended to use ts-ls or deno-ls instead.")))))
 
+
+
```

Comment:
What's wrong with all these extra newlines? 🤔

Reply:
They are page-break characters `^L`, not new line.
I've separated each server into a page